src/videojs.ads.js

```diff
@@ -246,6 +268,37 @@ var
       // just resume playback at the current time.
       player.play();
     }
+
+    i = suppressedTracks.list.length;
+    if (tracks.addTrack_) {
+      while (i--) {
+        track = suppressedTracks.list[i];
+        tracks.addTrack_(track);
```

Comment:
How about snapshotting the cues? That avoids the double request/parse and their are public APIs for recreating them.